patches/@types/draft-js+0.10.24.patch

```diff
@@ -0,0 +1,19 @@
+patch-package
```

Comment:
Do we need this file?

Reply:
@ioanpocol I added a new dependency: `patch-package`. With it we can modify any package under node_modules and make a patch like this one to fix a bug without waiting [for a PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28620) to be merged. This will patch the module after an `npm install` automatically

Reply:
👍